packages/vx-responsive/src/enhancers/withParentSize.js

```diff
@@ -28,7 +28,7 @@ export default function withParentSize(BaseComponent) {
 
     resize(event) {
       if (this.container) {
-        var boundingRect = this.container.getBoundingClientRect();
+        const boundingRect = this.container.getBoundingClientRect();
```

Comment:
I think you're right about the scope, played with it for a while on babel + locally. I remembered having an issue with something similar in a different repo but I think the root cause was something else. thanks for the SE link 👍